modules/images/src/image-loader.js

```diff
@@ -13,7 +8,7 @@ export default {
   name: 'Images',
   extensions: EXTENSIONS,
   parse: canParseImage && parseImage,
-  loadAndParse: !canParseImage && loadImage
+  loadAndParse: typeof Image !== 'undefined' && loadToHTMLImage
```

Comment:
Why this change? This no longer captures whether the node image parser has been installed?

Is `canParseImage` used at all after this change?

Reply:
It's still used when in web worker and node.